src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -406,6 +406,7 @@ def _handle_group_actions(
                     )
 
                 view = View(**slack_request.data["view"])
+                assert view.private_metadata is not None
```

Comment:
it's really unclear why mypy suddenly noticed this as being nullable -- the type checker is probably (?) right that this _could_ be None but it would have been a crash either way before/after so I don't feel too bad adding the assertion here